### ISSUE TITLE
effective porosity

### ISSUE THREAD

**Issue by alessiofumagalli** [pmgbergen/porepy#153]
we should consider the effective porosity and not the porosity itself. this might have some other implications.

**Comment by rbe051** [comment]
Can you elaborate a bit more? I just remeber @IvarStefansson came back from London and though this was important

**Comment by keileg** [comment]
I'm not sure where you are headed with this. To me, porosity is a unitless quantity, which will have a similar definition in all dimensions, and I don't see right away what we can gain from introducing an effective version. 

If you are thinking of the volume of an embedded object (say a fracture in 3D), then for sure we need to be careful with the units; the natural choice to me is to take the measure of the cell, and multiply it with the cross-sectional area, or whichever word you want to use for this. The latter quantity is naturally associated with some notion of aperture, interpreted as a volumetric / geometric concept.

In summary, I agree with @rbe051 , please elaborate.

**Comment by alessiofumagalli** [comment]
No need to keep this issue alive anymore, it's not a code related thing but more a philosophical thing
